Documentation/ceph-csi-drivers.md

```diff
@@ -175,3 +175,68 @@ CSI-Addons supports the following operations:
   - [Annotating PersistentVolumeClaims](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.3.0/docs/reclaimspace.md#annotating-perstentvolumeclaims)
 - Network Fencing
   - [Creating a NetworkFence](https://github.com/csi-addons/kubernetes-csi-addons/blob/v0.3.0/docs/networkfence.md)
+
+### Enable RBD Encryption Support
```

Comment:
This isn't part of the CSI addons controller, right? If not, it can move up in the header level.
```suggestion
## Enable RBD Encryption Support
```